doc/sphinx/addendum/program.rst

```diff
@@ -295,6 +295,14 @@ optional tactic is replaced by the default one if not specified.
 
    Start the proof of the next unsolved obligation.
 
+.. cmd:: Final Obligation {? of @ident } {? with @ltac_expr }
```

Comment:
In `common.edit_mlg`, you need something like this to pass lint:

```
command: [
   :
| REPLACE "Final" "Obligation" "of" identref withtac
| WITH "Final" "Obligation" OPT ( "of" identref ) withtac
| DELETE "Final" "Obligation" withtac
```

Reply:
seems to pass fine